ci-operator/step-registry/hypershift/kubevirt/install/odf/hypershift-kubevirt-install-odf-ref.yaml

```diff
@@ -11,10 +11,13 @@ ref:
   env:
   - name: ODF_OPERATOR_CHANNEL
     documentation: The odf operator channel
-    default: "stable-4.13"
```

Comment:
i'm not sure what this will impact. It could end up breaking 4.14 ci. i'm not sure.